src/main/java/org/commcare/formplayer/services/CaseSearchHelper.java

```diff
@@ -36,27 +37,31 @@
 
     private final Log log = LogFactory.getLog(CaseSearchHelper.class);
 
-    public ExternalDataInstance getSearchDataInstance(FormplayerQueryScreen screen, URI uri) {
+    @Override
+    public ExternalDataInstance getRemoteDataInstance(String instanceId, boolean useCaseTemplate, URI uri)
+            throws UnfullfilledRequirementsException, XmlPullParserException, InvalidStructureException, IOException {
         Cache cache = cacheManager.getCache("case_search");
         String cacheKey = getCacheKey(uri);
         TreeElement cachedRoot = cache.get(cacheKey, TreeElement.class);
```

Comment:
not really related to this PR but how does this cache get cleared? When I was testing locally I wasn't able to get updated case list results because of this cache.